lms/djangoapps/branding/__init__.py

```diff
@@ -27,6 +33,14 @@ def get_visible_courses():
 
     filtered_by_org = microsite.get_value('course_org_filter')
 
+    filtered_by_db = TileConfiguration.objects.filter(
+        enabled=True,
+    ).values('course_id').order_by('-change_date')
+
+    if filtered_by_db:
+        filtered_by_db_ids = [course['course_id'] for course in filtered_by_db]
+        filtered_by_db_keys = frozenset([SlashSeparatedCourseKey.from_deprecated_string(c) for c in filtered_by_db_ids])
```

Comment:
This shouldn't assume that `CourseKeys` are always old-style triples. Since we're looking to enable split_mongo (and course_rereruns) soon, we'll need to make this OpaqueKeys-compliant and not refer to deprecated strings.